mapstory/static/style/site/site.less

```diff
@@ -1,6 +1,6 @@
 //fonts
-@import '../../fonts/lato_font.css';
 @import '../../site/font-awesome.css';
+@import 'https://fonts.googleapis.com/css?family=Lato:400,400i,700,900';
```

Comment:
The way we were importing Lato previously wasn't working. This uses the modern `@import` code from Google Fonts.